datarepo-clienttests/src/main/java/runner/TestRunner.java

```diff
@@ -181,9 +183,19 @@ private void executeTestConfigurationNoGuaranteedCleanup() throws Exception {
     for (TestScriptSpecification testScriptSpecification : config.testScripts) {
       TestScript testScriptInstance = testScriptSpecification.scriptClassInstance();
 
+      testScriptInstance.setCloudPlatform(
+          Optional.ofNullable(CloudPlatform.fromValue(config.cloudPlatform))
+              .orElse(CloudPlatform.GCP));
```

Comment:
You can avoid the hardcoding of the default `GCP` here by using `CloudPlatformWrapper`.
```suggestion
          CloudPlatformWrapper.of(config.cloudPlatform).getCloudPlatform());
```

Reply:
Oh, I forgot, this is clienttests so we can't do this. Seems like a poor setup since it leads to code duplication in the tests.